src/Documentation/Grain-Versioning/Compatible-grains.md

```diff
@@ -1,22 +1,50 @@
 # Compatible grains
 
-In this document will be described the notion of "backward compatible" and "fully compatible".
+When an existing grain activation is about to process a request, the runtime will check if the version
+in the request and the actual version of the grain are compatible.
+The default behavior to determine if two versions are compatible is determined by `GlobalConfiguration.CompatibilityStrategy`
 
-## Backward compatible
+## Backward compatible (default)
+
+### Definition
 
 A grain interface version Vn is said to be backward compatible with Vm if:
 
   - The name of the interface didn't change (or the overridden typecode)
   - All public methods present in the Vm version are in the Vn version. __It is important that
-    the signatures of the methods inherited from Vm are not modified__.
+    the signatures of the methods inherited from Vm are not modified__: since Orleans use
+    an internal built-in serializer, modifying/renaming a field (even private) can make the
```

Comment:
I would clarify that this is prescriptive guidance you should follow when using the Backward compatible policy. It is not clear from reading this that following these tips is separate from configuring the policy.
Reading this it might look like Orleans would infer at runtime which policy to use depending on the deployed code and how compatible it is from version to version.